cell_classification/segmentation_data_prep_test.py

```diff
@@ -33,9 +53,122 @@ def test_get_image():
         assert not np.array_equal(CD8_img, CD4_img)
 
 
-def test_prepare_example():
+def prepare_test_data_folders(num_folders, temp_dir, selected_markers, random=False, scale=1.0):
```

Comment:
If you make `scale` a list, or have `scale` linearly increase for each `marker` in `selected_markers`, you'll be able to test different normalization coefficients for different markers